trick_source/sim_services/MonteCarloGeneration/mc_variable_random_normal.cc

```diff
@@ -105,6 +106,19 @@ MonteCarloVariableRandomNormal::generate_assignment()
   assign_double(assignment_d);
 }
 
+/*****************************************************************************
+summarize_variable
+Purpose:(Provide a string summarizing the attributes of this MonteCarloVariable)
+*****************************************************************************/
+std::string MonteCarloVariableRandomNormal::summarize_variable() const
+{
+  std::ostringstream ss;
+  ss << MonteCarloVariable::summarize_variable() << std::string(" dispersion=Normal,")
+     << " mean=" << distribution.mean() << ", stddev=" << distribution.stddev()
+     << " min_value=" << min_value << ", max_value=" << max_value
```

Comment:
missing a comma between stddev and min_value

Reply:
Nice catch, will fix!